### ISSUE TITLE
Request for more examples

### ISSUE THREAD

**Issue by atselvan** [go-ldap/ldap#150]
Hi,

Could you please add more example on adding entries, deleting entries, adding a user to a group, etc.

Similar to:

Examples
Conn.Bind
Conn.Compare
Conn.Modify
Conn.PasswordModify (Admin)
Conn.PasswordModify (GeneratedPassword)
Conn.PasswordModify (SetNewPassword)
Conn.Search
Conn.StartTLS
package (Beherappolicy)
package (UserAuthentication)
package (Vchuppolicy)

Regards,
Allan Selvan

**Comment by johnweldon** [comment]
Agreed, this would be nice.  Feel free to suggest some examples in a PR.

**Comment by 4cq2** [comment]
https://github.com/go-ldap/ldap/issues/444

**Comment by p0dalirius** [comment]
Hi,

I am trying to provide a minimum working example on connecting to a Windows Server's LDAP using Kerberos (https://github.com/go-ldap/ldap/issues/536). Hopefully once it works it will be used as an example in the docs.
